src/tests/test_dynamic_api/test_router.py

```diff
@@ -29,3 +29,66 @@ def test_reload_delete(router, bommen_dataset):
 
     with pytest.raises(NoReverseMatch):
         assert reverse("dynamic_api:bommen-bommen-list")
+
+
+@pytest.mark.django_db
+def test_only_selected_datasets_loaded(
+    settings, router, bommen_dataset, gebieden_dataset, meldingen_dataset
+):
+    router.reload()
+    assert reverse(
+        "dynamic_api:bommen-bommen-list"
+    )  # Bommen dataset reverse works, as dataset is registered
```

Comment:
misschien een comment boven de code plaatsen ivm black changes

Reply:
Oh yes, forgot to check after black did black things.